tests/_tests_files/test_integrator_wire_navigation.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?>
+<config><before-main><label>Before main</label><title>Before main</title><icon>fa-archive</icon><bundle>before-main</bundle><controller>index</controller><action>index</action><pages><before-main-nested><label>Before main - nested</label><title>Before main - nested</title><icon>fa-archive</icon><bundle>before-main-nested</bundle><controller>index</controller><action>index</action><visible>0</visible></before-main-nested></pages></before-main><main><label>Main</label><title>Main</title><icon>fa-archive</icon><bundle>main</bundle><controller>index</controller><action>index</action></main><after-main><label>After main</label><title>After main</title><icon>fa-archive</icon><bundle>after-main</bundle><controller>index</controller><action>index</action><visible>0</visible></after-main><new-latest><label>New latest</label><title>New latest</title><icon>fa-archive</icon><bundle>after-main</bundle><controller>index</controller><action>index</action><visible>0</visible></new-latest></config>
```

Comment:
Is this the expected result for testing? If so take a look at the formatting.

Reply:
do we have an option to add formatted xml code using this library?

Reply:
I leave it as it is because of its comment https://github.com/spryker-sdk/integrator/pull/15#discussion_r810911505
I will check if somehow can change it.

Reply:
I fixed it but in the result file we have 2 spaces in one tab instead of 4. Is it Ok? Or we should read Schema as a string and execute replace?

Reply:
The comment you linked also shows an easy way of how to format this file more properly. Can't you use the same approach?

Reply:
Thank you, missed it. Already fixed it.